benchmark.py

```diff
@@ -332,6 +332,32 @@ def _run_service_test(example, service, test_suite_config, namespace):
         else:
             print(f"[OPEA BENCHMARK] 🚀 Dataset is not specified for {service_name}. Check the benchmark.yaml again.")
 
+        bench_target = test_suite_config.get("bench_target")[0]
+
+        # If benchmark target if docsum then add dataset to run yaml file
+        if bench_target in ["docsumbench", "docsumfixed"]:
+            run_yaml_data = {}
+            # Open and read the YAML file
+            with open(run_yaml_path, "r") as file:
+                try:
+                    run_yaml_data = yaml.safe_load(file)
+                except yaml.YAMLError as exc:
+                    print(f"Error reading YAML file: {exc}")
+                    exit(1)
+
+            # Modify the dataset values
+            try:
```

Comment:
But I think better to handle and write the dataset into config yaml here, https://github.com/opea-project/GenAIExamples/pull/2070/files#diff-95c886c4c3a64881051e8eb5604d8ba7364b3094226313d752fe2e3c9ab4eaa1R169